vars/helm.groovy

```diff
@@ -53,6 +53,8 @@ String generateModuleValues(String module_name, String module_version, Project p
         String repository
         if (custom_module || module_name == 'ui-bundle') {
             repository = Constants.ECR_FOLIO_REPOSITORY
+        } else if (module_name == 'mod-graphql') {
```

Comment:
and what if module version will be release? For example: this part will works only if mod-graphql-1.2.400000026 but if it will be mod-graphql-1.2.4 then it will fails